src/redux/features/CreateReviewSlice.ts

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { PayloadAction, createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { DynamicData } from '../../@types/DynamicData';
+import API from '../../utils/api';
+
+interface reviewType {
+	productId: string;
+	feedBack: string;
+	ratings: number;
+	token: string;
+}
+
+const initialState: DynamicData = {
+	isLoading: false,
+	data: [],
+	error: null,
+};
+
+export const createReview = createAsyncThunk(
+	'createReview',
+	async (
+		{ productId, feedBack, ratings, token }: reviewType,
+		{ rejectWithValue },
+	) => {
+		try {
+			const { data } = await API.post(
+				'/reviews',
+				{ productId, feedBack, ratings },
+				{
+					headers: {
+						Authorization: `Bearer ${token}`,
+					},
```

Comment:
remove this headers, our exios interceptor has already configuration for handling this